cmd/jag/commands/util.go

```diff
@@ -31,6 +31,15 @@ type SDK struct {
 	Version string
 }
 
+func printFixLine() {
```

Comment:
I think for now it's easier if the `printFixLine` simply returns the string and then uses it in the `fmt.Errorf`.